### DISCUSSION TITLE
A probable mistake in a guideline from the website

### DISCUSSION THREAD

**Discussion by kainio** [saleor/saleor#10127] [General] [answered]
In this page
https://saleor.io/open-source/

those two commands are switched I think.

# Finally, create yourself an admin account:
docker-compose run --rm api python3 manage.py populatedb
# Populate the database with sample data and create the admin user:
docker-compose run --rm api python3 manage.py createsuperuser

**Comment by karolkielecki** [comment, marked as answer]
Hi @kainio, thanks for reporting! Indeed, there is a mistake over there. We'll update it as soon as we can. 

Cheers!
